src/kernels/raw/finder/localPythonAndRelatedNonPythonKernelSpecFinder.node.ts

```diff
@@ -70,12 +70,12 @@ export class LocalPythonAndRelatedNonPythonKernelSpecFinder extends LocalKernelS
         @inject(IPythonExtensionChecker) extensionChecker: IPythonExtensionChecker,
         @inject(LocalKnownPathKernelSpecFinder)
         private readonly kernelSpecsFromKnownLocations: LocalKnownPathKernelSpecFinder,
-        @inject(IMemento) @named(GLOBAL_MEMENTO) globalState: Memento,
+        @inject(IMemento) @named(WORKSPACE_MEMENTO) memento: Memento,
```

Comment:
ensure we store stuff in workspace memento, not global
as interpreters and the like are specific to a workspace